docs/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!-- Thank you for submitting this issue!
+-->
+
+**Is this a BUG REPORT or FEATURE REQUEST?**:
+
+<!-- Provide a more detailed introduction to this issue and why it's relevant. 
+-->
+**Description**:
+
+<!-- Explain what you would expect from this process.
+-->
+**Expected Behavior**:
+
+<!-- Describe what you're currently experiencing from this process, and thereby explain the bug.
+-->
+**Actual Behavior**:
+
+<!-- Please provide an ordered list on how to reproduce the issue.
+-->
+**Steps to Reproduce**:
+
+<!-- if relevant, please include a screenshot.
+-->
+**Screenshots (if any)**:
+
+**Would you like to work on this issue?**
```

Comment:
Add "Please let us know if you can work on it or the issue should be assigned to someone else." this in comment